readme/7.1/BREAKING_CHANGES.markdown

```diff
@@ -452,4 +452,40 @@ environment is a very serious security issue.
 A value of `true` will improve a portal administrator's experience, and a value
 of `false` can be considered during performance tuning, if needed.
 
+---------------------------------------
+
+### Users can have numeric screen names with no limitations, and sites can no longer have numeric friendly URLs
+- **Date:** 2017-Oct-10
+- **JIRA Ticket:** LPS-66460
+
+#### What changed?
+
+- The portal property `users.screen.name.allow.numeric` now defaults to **true**
+- Numeric screen names are no longer limited by whether or not they correspond to an existing group ID.
+- Groups are no longer allowed to set their group ID as their friendly URL
+- Groups can no longer be accessed by using their group ID in the URL (this used to be reconciled automatically, even if it wasn't set that way)
+- If a group is updated and no friendly URL is provided, the default friendly URL is now `/group-<groupId>`. If that is a duplicate of another friendly URL, the friendly URL will be incremented until a unique URL is found (such as `/group-<groupId>-1`)
+
+#### Who is affected?
+
+- Anyone who has set the friendly URL of their site to the group ID
+- Anyone who uses a group ID to navigate or direct to a site
+
+#### How should I update my code?
+
+No code updates should be required, but if you are in one of the scenarios from the "Who is affected?" section, you should consider the following changes:
+
+- If you have set the friendly URL of a site to its group ID, you must update the friendly URL of that site.  This can be done by the site administrator through the Site Settings portlet.
```

Comment:
This got me thinking what if the users fail to update the friendly URL of the sties that uses the group ID. Will they still be able to access the site by the groupID? If not, I think we should have an upgrade process to update the groupID friendly URLs to the default, `/group-<groupID>`.

Reply:
I have updated the document to reflect this, but an upgrade process should not be necessary.  If a site has a numeric friendlyURL, it will continue to work as normal after this change.  This is because we still check for group by groupId in the FriendlyURLServlet.  However, they will be forced to change it whenever they want to update the site through Site Settings.